### PR TITLE
Added Health facilities view to the Api

### PR DESCRIPTION
**Pull request by Awinja-j**
#### Description of Task to be completed:
The task required writing tests for health facilities and its end points.

#### Manual testing:
You can manually test it by running ```nosetests``` or ```python test_health_facilities.py```

#### Relevant Trello stories?
[HTools API] Add Health Facilities view

### CODE REVIEW

File: healthtools_ke_api/__init__.py [hunk 1]
```diff
@@ -16,6 +17,7 @@
 app.register_blueprint(doctors_api, url_prefix='/doctors')
 app.register_blueprint(nurses_api, url_prefix='/nurses')
 app.register_blueprint(clinical_officers_api, url_prefix='/clinical-officers')
+app.register_blueprint(health_facilities_api, url_prefix='/health_facilities')
```
Comment (DavidLemayian) [inline healthtools_ke_api/__init__.py]: This should be `health-facilities`. With a `-` instead of `_`.

File: healthtools_ke_api/views/search.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def remove_keyword(query):
         Remove keyword from search term
         """
         query_formatted = query.strip().lower()
-        keywords = ['dr', 'dr.', 'doctor', 'nurse', 'co', 'c.o.', 'c.o', 'clinical officer']
+        keywords = ['dr', 'dr.', 'doctor', 'nurse', 'co', 'c.o.', 'c.o', 'clinical officer','health facility']
```
Comment (DavidLemayian) [inline healthtools_ke_api/views/search.py]: What other derivatives for "health facilities" can we think about?

**Review by DavidLemayian — CHANGES_REQUESTED** [review]
Let's also git-ignore `.DS_Store`.

Does this work with SMS?

File: healthtools_ke_api/__init__.py (hunk 1 above)
Comment (Awinja-j) [reply healthtools_ke_api/__init__.py]: @DavidLemayian Changed.

File: healthtools_ke_api/views/search.py (hunk 2 above)
Comment (Awinja-j) [reply healthtools_ke_api/views/search.py]: @DavidLemayian hf and h.f.
I have added these two.

Comment (DavidLemayian) [reply healthtools_ke_api/views/search.py]: @Awinja-Andela Hmm.. I don't see them added here.. "facility" as well perhaps?

File: healthtools_ke_api/__init__.py (hunk 1 above)
Comment (DavidLemayian) [reply healthtools_ke_api/__init__.py]: @Awinja-Andela I can't see the change reflected on this PR..

File: healthtools_ke_api/views/search.py [hunk 3]
```diff
@@ -33,7 +33,7 @@ def remove_keyword(query):
         Remove keyword from search term
         """
         query_formatted = query.strip().lower()
-        keywords = ['dr', 'dr.', 'doctor', 'nurse', 'co', 'c.o.', 'c.o', 'clinical officer','health facility']
+        keywords = ['dr', 'dr.', 'doctor', 'nurse', 'co', 'c.o.', 'c.o','hf', 'h.f', 'h.f.', 'clinical officer','health facility']
```
Comment (DavidLemayian) [inline healthtools_ke_api/views/search.py]: Let's cluster these appropriately i.e `'clinical officers'` should come before the health facility alternatives. We don't simply do.. we do well.

File: manage.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
 from healthtools_ke_api import app
 
 if __name__ == "__main__":
-    app.run()
+    app.run(debug=True)
```
Comment (DavidLemayian) [inline manage.py]: This should be environment settable.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 5]
```diff
@@ -87,3 +87,4 @@ ENV/
 
 # Rope project settings
 .ropeproject
+.DS_Store
```
Comment (DavidLemayian) [inline .gitignore:90]: This should be deleted and gitignored from every folder.

Please make sure it doesn't come through to the PR.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

**Pull request closed without merging**